vm/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -6,7 +6,14 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
 Anypoint Connector for Virtual Machine (VM Connector) manages intra-app and inter-app communication through either transient or persistent asynchronous queues:
 
 * Transient queues are faster than persistent queues, but they are not reliable if the system crashes.
-* Persistent queues are slower but reliable.
+* Persistent queues are slower but reliable. +
++
+When running a Mule application on a single runtime instance mode, persistent queues work by serializing and storing the contents on the disk. However, when running the Mule application in cluster runtime instance mode, the persistent queues are backed up in the memory grid. Thereby, when a flow uses VM Connector to publish content to a queue, Mule runtime engine determines whether to process the message in the same origin node or to send the message to the cluster to pick another node. Using VM Connector facilitates the load distribution across a cluster. +
```

Comment:
```suggestion
When running a Mule application in single runtime instance mode, persistent queues work by serializing and storing the contents on the disk. However, when running the Mule application in cluster runtime instance mode, the persistent queues are backed up in the memory grid. Thereby, when a flow uses VM Connector to publish content to a queue, Mule runtime engine determines whether to process the message in the same origin node or to send the message to the cluster to pick another node. Using VM Connector facilitates the load distribution across a cluster. +
```

Reply:
Not following what you recommended here. Is the extra line at the beginning?